celery/concurrency/asynpool.py

```diff
@@ -205,6 +205,59 @@ def _select(readers=None, writers=None, err=None, timeout=0,
             raise
 
 
+try:  # Python 2 does not have FileNotFoundError
```

Comment:
Would you mind adding a TODO to drop this once we drop support for Python 2.7?

Reply:
Done!